### PR TITLE
PIM-7251: Fix history date on all grids

### PR DESCRIPTION
**Pull request by Doodoune**
**Description**

Since 2.0, no more update date on grids except product form
image

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | -
| Added legacy Behats               | -
| Added acceptance tests            | -
| Added integration tests           | -
| Changelog updated                 | OK
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | Todo
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: src/Pim/Bundle/DataGridBundle/Extension/Formatter/Property/ProductValue/DateProperty.php [hunk 1]
```diff
@@ -37,4 +38,19 @@ protected function convertValue($value)
 
         return $this->presenter->present($result, ['locale' => $this->translator->getLocale()]);
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function getRawValue(ResultRecordInterface $record)
```
Comment (skeleton) [inline src/Pim/Bundle/DataGridBundle/Extension/Formatter/Property/ProductValue/DateProperty.php]: if it's work on product grid, why your fix is in namespace ProductValue? this class concerns all values related to product normally

**Comment by phaseinducer** [comment]
Adding a little test could be a good idea. It was a regression already, let's avoid it next time.

File: src/Pim/Bundle/DataGridBundle/Extension/Formatter/Property/ProductValue/DateProperty.php (hunk 1 above)
Comment (Doodoune) [reply src/Pim/Bundle/DataGridBundle/Extension/Formatter/Property/ProductValue/DateProperty.php]: On history.yml, the value "loggedAt" : 
            loggedAt:
                label: Logged at
                type: product_value_datetime
                frontend_type: datetime
It came from this configuration ... Maybe initially it was only on Product grid.

**Comment by Doodoune** [comment]
@phaseinducer - some tests added for all history tab

File: src/Pim/Bundle/DataGridBundle/Extension/Formatter/Property/ProductValue/DateProperty.php [hunk 2]
```diff
@@ -3,6 +3,7 @@
 namespace Pim\Bundle\DataGridBundle\Extension\Formatter\Property\ProductValue;
 
 use Akeneo\Component\Localization\Presenter\PresenterInterface;
+use Oro\Bundle\DataGridBundle\Datasource\ResultRecordInterface;
```
Comment (damien-carcel) [inline src/Pim/Bundle/DataGridBundle/Extension/Formatter/Property/ProductValue/DateProperty.php]: Why do you need this `use` when it is not in the class?

File: src/Pim/Bundle/DataGridBundle/Extension/Formatter/Property/ProductValue/DateProperty.php (hunk 1 above)
Comment (Doodoune) [reply src/Pim/Bundle/DataGridBundle/Extension/Formatter/Property/ProductValue/DateProperty.php]: You're right, I change it ;)

File: src/Pim/Bundle/DataGridBundle/Resources/config/formatters.yml [hunk 3]
```diff
@@ -31,13 +31,13 @@ services:
         tags:
             - { name: oro_datagrid.extension.formatter.property, type: product_value_date }
 
-    pim_datagrid.extension.formatter.property.product_value.datetime_property:
-        class: '%pim_datagrid.extension.formatter.property.product_value.date_property.class%'
+    pim_datagrid.extension.formatter.property.datetime_property:
```
Comment (damien-carcel) [inline src/Pim/Bundle/DataGridBundle/Resources/config/formatters.yml]: Service renamed = BC break.
Please don't, and only add a comment for it to be renamed on master

File: src/Pim/Bundle/DataGridBundle/Extension/Formatter/Property/ProductValue/DateProperty.php (hunk 2 above)
Comment (Doodoune) [reply src/Pim/Bundle/DataGridBundle/Extension/Formatter/Property/ProductValue/DateProperty.php]: Oops ^^' I forget it from my previous fix for this ticket

File: src/Pim/Bundle/DataGridBundle/Resources/config/formatters.yml (hunk 3 above)
Comment (Doodoune) [reply src/Pim/Bundle/DataGridBundle/Resources/config/formatters.yml]: Okay

**Pull request merged**